data/categories.txt

```diff
@@ -9618,7 +9618,7 @@ cs:4bezpečnost
 de:Polizeiwache|Polizei
 fr:Poste de police|Commissariat
 hi:3थाना
-it:Posto di polizia
+it:Posto di polizia|6Commissariato|5Questura|4Caserma|4Guardia|5Carabinieri
```

Comment:
```suggestion
it:6Commissariato|5Questura|4Caserma|4Guardia|5Carabinieri
```
`Polizia` is present under the `@category_police` already and Posto is too generic term IMHO